README.md

```diff
@@ -261,6 +261,12 @@ In this part of the lab we'll add everyone's names and GitHub usernames to the
 repository so we'll be able to figure out who GitHub user `MightyWombat259`
 is in real life.
 
+> :exclamation: :mage_woman: **Faculty** need to make sure that
+> [branch protection is turned _off_](FACULTY_BRANCH_PROTECTION_SETTINGS.md)
+> for this first part of the lab. If
+> you don't then students won't be able to commit directly to `main`, which
```

Comment:
This is not true. A branch protection rule with none of the checks enabled just prevents _force_ pushes and deleting the branch. These are probably things we actually want.

Reply:
Thanks for catching that. I'll go back and fix that (again). 😓